### PR TITLE
Bug 1626987 - Use a single bundled tool for rendering the docs

### PR DESCRIPTION
**Pull request by badboy**
This is not perfect yet, as it still recommends doing it manually (download it and put it somehwere) or using a tool that requires Rust.

I will file follow-up bugs to tackle that as well as moving mdbook-dtmo under the mozilla org.

### CODE REVIEW

**Comment by badboy** [comment]
Lintcheck failed once because of spurious network errors.
And the spellcheck task was about to compile its own nodejs (including v8) because of (probably the same) network errros and failing to download the binary version of node....

I rekicked all tests now.

File: src/meta/contributing.md [hunk 1]
```diff
@@ -21,38 +21,36 @@ To begin contributing to the docs, fork the `firefox-data-docs` repo.
 ## Building the Documentation
 
 The documentation is rendered with [mdBook](https://github.com/rust-lang/mdBook).
-
-To build the documentation locally, you'll need additional preprocessors:
+We use a fork named [mdbook-dtmo][] that includes multiple different preprocessors:
```
Comment (wlach) [inline src/meta/contributing.md:24]:
```suggestion
We use a fork named [mdbook-dtmo] that includes a number of custom additions to mdbook for our environment (for example, a plugin to automatically generate a table-of-contents).
```

I'm pretty sure you don't need the extra `[]`.

File: src/meta/contributing.md [hunk 2]
```diff
@@ -21,38 +21,36 @@ To begin contributing to the docs, fork the `firefox-data-docs` repo.
...
 
 * [mdbook-toc](https://github.com/badboy/mdbook-toc/releases)
 * [mdbook-mermaid](https://github.com/badboy/mdbook-mermaid/releases)
 * [mdbook-open-on-gh](https://github.com/badboy/mdbook-open-on-gh/releases)
 
-Download releases for your system, unpack it and place the binary in a directory of your `$PATH`.
+You can download [mdbook-dtmo][] on the [Releases page][releases] for your system.
```
Comment (wlach) [inline src/meta/contributing.md:30]:
```suggestion
You can download [mdbook-dtmo] on the [github releases page](https://github.com/badboy/mdbook-dtmo/releases). Please use the latest version.
```

File: src/meta/contributing.md [hunk 3]
```diff
@@ -21,38 +21,36 @@ To begin contributing to the docs, fork the `firefox-data-docs` repo.
...
+Unpack it and place the binary in a directory of your `$PATH`.
+
+[mdbook-dtmo]: https://github.com/badboy/mdbook-dtmo
+[releases]: https://github.com/badboy/mdbook-dtmo/releases
```
Comment (wlach) [inline src/meta/contributing.md:34]:
```suggestion
```

File: src/meta/contributing.md [hunk 4]
```diff
@@ -21,38 +21,36 @@ To begin contributing to the docs, fork the `firefox-data-docs` repo.
 ## Building the Documentation
 
 The documentation is rendered with [mdBook](https://github.com/rust-lang/mdBook).
-
-To build the documentation locally, you'll need additional preprocessors:
+We use a fork named [mdbook-dtmo][] that includes multiple different preprocessors:
 
 * [mdbook-toc](https://github.com/badboy/mdbook-toc/releases)
 * [mdbook-mermaid](https://github.com/badboy/mdbook-mermaid/releases)
 * [mdbook-open-on-gh](https://github.com/badboy/mdbook-open-on-gh/releases)
```
Comment (wlach) [inline src/meta/contributing.md:28]: Let's just take these out -- we can just keep track of this information in mdbook-dtmo and won't have to remember to update this.

**Pull request merged**